src/pages/OperationsPage.js

```diff
@@ -47,10 +48,12 @@ class OperationsPage extends Component {
     }
 
     componentWillMount(){
-        this.props.getConfirmationsOnce();
-        this.props.getListCBE();
-        this.props.getPendingsOnce();
-        this.props.getLOCsOnce();
+        this.props.getConfirmationsOnce();   //  todo
+        this.props.getListCBE();   //  todo
+        this.props.getPendingsOnce();   //  todo
+        if (this.props.isLOCsNeedReload) {
```

Comment:
@Pavel-DV, if you don't agree - offer something. We can rename it to isListLoaded, but not in all cases we need to initialize only list, it could be something else too. I've explained you above why 'loaded' isn't suitable and why 'ready' is more suitable. I'm just want to find the best common name for each of such cases.
If you're going to rename 'ready' to 'isReady' (I think we should use 'is' for all flags) please rename it another components too, so we'll have common standardized implementation in all parts of application.